src/components/contentful/contentfulBlogPost/contentfulBlogPostCard.js

```diff
@@ -20,7 +20,7 @@ const ContentfulBlogPostCard = ({ content }) => {
       <div className="card">
         <Background
           color={blogPostBackgroundColor}
-          backgroundStyle={blogPostBackgroundStyle}
```

Comment:
It'll be great to also remove `blogPostBackgroundStyle` from the `const` definition, since it's no longer needed : )